src/components/Question.jsx

```diff
@@ -119,25 +119,46 @@ export default class Question extends React.Component {
     this.setState({ showIcon: false });
   };
 
-  render() {
+  _getHeaderClassName = () => {
     const question = this.props.question;
-    const subQuestion = question[Constants.HAS_SUBQUESTION];
+    const collapsible = this.props.collapsible;
     const options = this.context.options;
-    const questionComponent = this.renderQuestion(question);
-    if (FormUtils.isHidden(question)) {
-      return null;
-    }
+    const subQuestion = question[Constants.HAS_SUBQUESTION];
+
+    let headerClassName = [];
+
+    headerClassName.push(
+      Question.getEmphasizedOnRelevantClass(question),
+      "question-header"
+    );
+
+    if (FormUtils.isWizardStep(question))
+      headerClassName.push("wizard-step bg-primary text-white");
+    if (FormUtils.isEmphasised(question))
+      headerClassName.push(Question.getEmphasizedClass(question));
+    if (FormUtils.isSection(question))
+      headerClassName.push("section-background");
+    if (collapsible) headerClassName.push("cursor-pointer");
     if (
-      !FormUtils.isRelevant(question) &&
-      (this.context.options.debugMode ||
+      FormUtils.isRelevant(question) &&
+      (options.debugMode ||
         JsonLdObjectUtils.checkId(subQuestion, options.startingQuestionId))
     ) {
-      return <div className="show-irrelevant">{questionComponent}</div>;
+      headerClassName.push("show-irrelevant");
+    }
+
+    return headerClassName;
+  };
+
+  render() {
+    const question = this.props.question;
+    const questionComponent = this.renderQuestion(question);
+    if (FormUtils.isHidden(question)) {
+      return null;
     }
     if (!FormUtils.isRelevant(question)) {
```

Comment:
I don't see how this can work for irrelevant questions in debug mode.

Reply:
![image](https://user-images.githubusercontent.com/3962237/186478633-49eaaea4-715d-440f-ae59-ced1fc11e0c4.png)

Reply:
@LaChope I have tested this and it seems it is fixed now and works correctly if it is also tested, lets resolve it.

Reply:
it is tested automatically, resolving.